CRMImporter/Converters/StringToBool.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xrm.Sdk;
+
+namespace CRMImporter.Converters
+{
+    public class StringToBool : IConverter
+    {
+        public StringToBool(string trueValue, string falseValue = null)
+        {
+            this.TrueValue = trueValue.ToLower();
+            this.FalseValue = falseValue == null ? null : falseValue.ToLower();
+        }
+
+        private readonly string TrueValue;
+        private readonly string FalseValue;
+
+        public object Convert(object input, IOrganizationService service)

```

Comment:
This method would benefit from unit tests, especially because of the somewhat complex if check on line 24.